src/components/Card/Card.jsx

```diff
@@ -44,13 +44,15 @@ class Card extends React.Component {
       data,
       isExpanded,
       borderType,
+      borderColorRed,
```

Comment:
this should be borderColor, similar to borderType, so that we don't need to add other props later like borderColorGreen, borderColorBlue and so on. Ideally, it'd just be something like borderStyle (for even more flexibility) but that would be backward-incompatible so we can revisit it later.